doc/developers/advanced_installation.rst

```diff
@@ -26,7 +26,7 @@ Installing a nightly build is the quickest way to:
 
 ::
 
-  pip install --pre -f https://sklearn-nightly.scdn8.secure.raxcdn.com scikit-learn
+  pip install --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn
```

Comment:
oh wow, is it so slow? It's coming at a 35 kB/s for mi

Reply:
It's on your end. From here I got 2.2 MB/s.

Reply:
It would be weird for this to be slower from within the anaconda offices :)

Reply:
I work mostly from home :D but it's a 50 mb/s connection otherwise. Since it's the nightly, I don't think it's that important anyway :)